### ISSUE TITLE
Refresh DOM

### ISSUE THREAD

**Issue by jdonnell01** [carlganz/rintrojs#44]
Hello,

I am using rintrojs with my R Shiny application and there are DOM elements that appear based on user selections. I am able to simulate the user clicks during the intro, however the intro fails to navigate to the newly created elements. When I close the intro, re-open, and go through the steps again rintrojs is able to now navigate to the new elements (I would guess that a snapshot of the DOM is captured when the intro begins). I attempted to call a javascript function that runs introJs().refresh() (https://introjs.com/docs/hints/api/#introjsrefresh) when a step is triggered, however this did not work. Is there a function within this API that refreshes the DOM?

Thank you

**Comment by carlganz** [comment]
Yeah dynamic content and `intro.js` are hard to work with. I honestly wasn't ware of the refresh function in `intro.js`

If you can find a way to use the refresh function with dynamic content in a vanilla javascript example (ie don't use Shiny) there is a good chance we can make it work with Shiny and `rintrojs`
